README.md

```diff
@@ -24,10 +24,15 @@ extensible to working with different manifest schemas and PKI tooling.
 
 ## Public Instance
 
-We're currently working hard on cutting a 1.0 release and productionizing the public instance.
-We don't have a date yet, but follow along on the [GitHub project](https://github.com/orgs/sigstore/projects/5).
+Rekor is officially Generally Available with a 1.0 release.
```

Comment:
perhaps we should explicitly acknowledge that we intend to adhere to https://semver.org/ rules around API stability?

Reply:
sure, added